packages/eslint-plugin/src/rules/no-empty-function.ts

```diff
@@ -48,7 +48,9 @@ export default util.createRule<Options, MessageIds>({
       extendsBaseRule: true,
     },
     schema: [schema],
-    messages: baseRule.meta.messages,
+    messages: baseRule.meta.messages ?? {
+      unexpected: 'Unexpected empty {{name}}.',
+    },
```

Comment:
this rule has been using message ids since 4.17, so we don't need to fallback here.

Reply:
reverted this change